src/main/java/com/glovo/ttglovo/securityManagement/appuser/AppUser.java

```diff
@@ -0,0 +1,172 @@
+package com.glovo.ttglovo.securityManagement.appuser;
+
+import com.glovo.ttglovo.cart.CartItem;
+import lombok.*;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+import java.util.*;
+
+@Builder
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name = "AppUser")
+@Table(name = "user_table",
+        uniqueConstraints = {
+                @UniqueConstraint(
+                        columnNames = "email",
+                        name = "uk_email"
```

Comment:
With unique constraints it is common to add "uniq" to the end of the name -> _"uk_email_uniq"_